### PR TITLE
chore(deps): update dependencies and bump release

### PR DESCRIPTION
**Pull request by kalinchernev**
Bumps package version to latest release. It's not +1 since a merge of #77 happened later than other releases in the meantime.

Updates package to latest versions. Most importantly, we have to upgrade commander to avoid this [security issue](https://github.com/tj/node-growl/issues/60).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,13 +16,7 @@
...
     "url": "https://github.com/Surnet/swagger-jsdoc.git"
   },
-  "keywords": [
-    "jsdoc",
-    "restful",
-    "api",
-    "express",
-    "swagger"
-  ],
+  "keywords": ["jsdoc", "restful", "api", "express", "swagger"],
```
Comment (drGrove) [inline package.json]: Can we keep new lines between items in array?

Comment (kalinchernev) [inline package.json]: Hi @drGrove I reverted both changes.

**Comment by drGrove** [comment]
Circle CI looks to be failing due to the version of node we're running vs the version of node expected for babylon.

**Comment by kalinchernev** [comment]
Unfortunetely the build is failing, and I can't do anything about it because I don't have sufficient rights to do anything in CirlceCI. It's also normal for the version of node to be a questionable part of the stack. I previously touched both topics in #76
The situation is a reason for me to stop taking actions on this repo since I can't really fix issues like these.

**Comment by chdanielmueller** [comment]
Hi @kalinchernev 
You can change the CircleCI config in https://github.com/Surnet/swagger-jsdoc/blob/master/circle.yml
What do you need me to do? I would be glad to help you.
Thanks,
Daniel

**Comment by kalinchernev** [comment]
Hi @chdanielmueller the master seems ok, thanks for following up here.
I haven't had the time to investigate the new circle ci topics, but most importantly the master seems stable. Please don't forget to publish the latest package to npm and the releases :)

**Comment by drGrove** [comment]
@kalinchernev @chdanielmueller publishing v1.9.4 now

**Comment by kalinchernev** [comment]
It will have to be 1.9.7. After this, you'll have to sync the master of the repo as [before](https://github.com/Surnet/swagger-jsdoc/commit/0752561610ada57430204df88f8c043c8362ab67). Preferably, each release on npm has to be matched as a release in the repo. The timeline shows [some examples](https://github.com/Surnet/swagger-jsdoc/commits/master?after=ed21dc988f801d302f7fa6381a2797fdaab7f60a+34) which worked before the master was locked. Last time I tried, I couldn't make this follow-up on the master here. (The time I tried to keep a changelog)

**Pull request merged**